### PR TITLE
feat(puppeteer): update for 1.10

### PR DESCRIPTION
**Pull request by SimonSchick**
Please fill in this template.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/GoogleChrome/puppeteer/releases/tag/v1.10.0
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/puppeteer/index.d.ts [hunk 1]
```diff
@@ -1331,6 +1327,137 @@ export interface GeoOptions {
 
 export type MediaType = "screen" | "print";
 
+export interface SnapshopOptions {
+/**
```
Comment (marvinhagemeister) [inline types/puppeteer/index.d.ts]: nit: indentation is off here

Comment (marvinhagemeister) [reply types/puppeteer/index.d.ts]: And I'd move this interface down to right above the function declaration where it is used.

**Review by marvinhagemeister — APPROVED** [review]
That's really awesome, thanks for making a PR 👍 🎉 

Just verified the changes against their API docs and can confirm that everything is correct 👍 I've made two (really) minor nit comments regarding formatting. But these are more my opinion and don't change anything functionality wise.

Thanks again for your contribution, you rock 👍 💯

**Comment by SimonSchick** [comment]
Thanks, I wish the puppeteer team would ship their own typings tho, the API is fairly complex :P

**Pull request merged**